rustls/Cargo.toml

```diff
@@ -23,8 +23,8 @@ log = { version = "0.4.4", optional = true }
 once_cell = { version = "1.16", default-features = false, features = ["alloc", "race"] }
 ring = { version = "0.17", optional = true }
 subtle = { version = "2.5.0", default-features = false }
-webpki = { package = "rustls-webpki", version = "0.102.2", features = ["alloc"], default-features = false }
-pki-types = { package = "rustls-pki-types", version = "1.2", features = ["alloc"] }
+webpki = { package = "rustls-webpki", version = "0.102.4", features = ["alloc"], default-features = false }
+pki-types = { package = "rustls-pki-types", version = "1.7", features = ["alloc"] }
```

Comment:
Since https://github.com/rustls/rustls/pull/1922 was merged first I think you can drop this part of your diff now :+1: